spec/models/user_option_spec.rb

```diff
@@ -44,6 +44,30 @@
       expect(user.user_option.email_digests).to eq(false)
       expect(user.user_option.digest_after_minutes).to eq(0)
     end
+
+    it "should correctly set sidebar_link_to_filtered_list when enabled" do
```

Comment:
```suggestion
    it "should correctly set sidebar_link_to_filtered_list when `default_sidebar_link_to_filtered_list` site setting is enabled" do
```

Small one but I think we can be clearer with the description here and in the other cases.